styles/main.css

```diff
@@ -2445,3 +2445,6 @@ blockquote p:last-child {
     font-size: 24px;
   }
 }
+.overflowHidden {
+  overflow: hidden !important;
+}
```

Comment:
@ntorbinskiy 

Why you are breaking CSS convention again?
Did you ever see somebody naming classes like that using camelCase in CSS files?

Where is the break line between classes?

Why did you put this line of code to the end of file?

Reply:
@ntorbinskiy I just did the check, it works without important, why do you use important statement then?